utils/writeable/src/cmp.rs

```diff
@@ -6,7 +6,7 @@ use core::cmp::Ordering;
 use core::fmt;
 
 pub(crate) struct WriteComparator<'a> {
-    string: &'a [u8],
+    code_points: &'a [u8],
```

Comment:
Nit: I think I prefer this to be `string`, but did you mean `code_units` instead of `code_points`?

Reply:
ah yes, `code_units`